vue/src/models/rop_store.js

```diff
@@ -28,6 +28,9 @@ Vue.use(Vuex);
 Vue.use(VueResource);
 Vue.use(VueLogger, logOptions);
 
+
+// In the futures, this should be replaced with Pina library.
```

Comment:
we can create an issue for this, else there is a high chance that this will be forgotten.